TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -69,11 +75,14 @@ class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
     }
 
     private fun startNoteFragment(position: Int) {
-        val note = arrayAdapter?.getItem(position)
         setSelectedItem(position)
 
-        note?.let {
+        arrayAdapter?.getItem(position)?.let { inode ->
             activity?.let { activity ->
+                var note = loadNote("", "/n" + inode.id)
+                if (note == null)
+                    note = Note(inode.id)
```

Comment:
this is just a question (no change required unless a comment would be helpful), but what is the purpose of setting it to the inode id field if it doesn't load?

Reply:
if you have an inode, but can't find an associated note (whether it's not there or invalid or something), this instantiates a new note for that inode